Modules/System/MicrosoftGraph/src/helper/MgOperationResponse.Codeunit.al

```diff
@@ -0,0 +1,97 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+codeunit 9138 "Mg Operation Response"
+{
+    Access = Internal;
+
+    [NonDebuggable]
+    [TryFunction]
+    internal procedure GetResultAsText(var Result: Text);
```

Comment:
I'm still not totally sure about the design of the Client if I want to return a boolean value for success or if it would be better to return the MgOperationResponse..

What are your thoughts on this topic?